scvi/model/_destvi.py

```diff
@@ -0,0 +1,282 @@
+import logging
+from typing import Optional, OrderedDict, Union
+
+import numpy as np
+import pandas as pd
+import torch
+from anndata import AnnData
+
+from scvi.data import register_tensor_from_anndata
+from scvi.model import CondSCVI
+from scvi.model.base import BaseModelClass, UnsupervisedTrainingMixin
+from scvi.module import MRDeconv
+
+logger = logging.getLogger(__name__)
+
+
+class DestVI(UnsupervisedTrainingMixin, BaseModelClass):
+    """
+    Multi-resolution deconvolution of Spatial Transcriptomics data (DestVI).
+
+    Parameters
```

Comment:
add a paragraph in the middle explaining that most users will initialize using from rna model (see stereoscope docstring)